build/bower.bundle.js

```diff
@@ -1868,7 +1890,11 @@ HotelSearchClient.prototype = {
     this._mergeStaticData(response);
     this._mergeHotels(response.hotels);
     this._mergeFilter(response.filter);
-    this._mergeRates(response.rates);
+    if (response.done || response.isLastPolling) {
```

Comment:
could we name `_mergeRates` to likes `_notLastMergeRates` and still use `this._mergeRates(response.rates)` here. Inside `_mergeRates` function, we do:

```_mergeRates: function(newRates) {
   if (response.done || response.isLastPolling) {
      this._lastMergeRates(response.rates);
    } else {
      this._notLastMergeRates(response.rates);  
    }
}```